app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/composables/Pills.kt

```diff
@@ -0,0 +1,103 @@
+package com.hedvig.app.feature.home.ui.claimstatus.composables
+
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.intl.Locale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hedvig.app.feature.home.ui.claimstatus.data.PillData
+import com.hedvig.app.ui.compose.theme.HedvigTheme
+import com.hedvig.app.ui.compose.theme.hedvigContentColorFor
+
+@Composable
+fun Pills(pillData: List<PillData>) {
+    Row(
+        modifier = Modifier.fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.Top,
+    ) {
+        Row(horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+            pillData.forEach { pillData: PillData ->
+                Pill(
+                    text = pillData.text,
+                    pillType = pillData.type,
+                )
+            }
+        }
+        // Uncomment when the card is clickable to go to detail screen
```

Comment:
```suggestion
        // TODO: Uncomment when the card is clickable to go to detail screen
```

Just so that we get some highlighting on it 😉